x-pack/plugins/security_solution/public/timelines/components/timeline/data_providers/index.tsx

```diff
@@ -84,7 +89,36 @@ const getDroppableId = (id: string): string =>
  * the user to drop anything with a facet count into
  * the data pro section.
  */
+
+const timelineSelectModeItemsClassName = 'timelineSelectModeItemsClassName';
+
+const searchOrFilterPopoverClassName = 'searchOrFilterPopover';
+const searchOrFilterPopoverWidth = '352px';
+
+const SearchOrFilterGlobalStyle = createGlobalStyle`
+  .${timelineSelectModeItemsClassName} {
+    width: 350px !important;
+  }
+
+  .${searchOrFilterPopoverClassName}.euiPopover__panel {
+    width: ${searchOrFilterPopoverWidth} !important;
+
+    .euiSuperSelect__listbox {
+      width: ${searchOrFilterPopoverWidth} !important;
+    }
+  }
+
+  [data-test-subj="timeline-select-search-or-filter"] {
```

Comment:
NIT: We should use `data-test-subj` for testing only, can we use a `class` or `id` for this style?